.github/workflows/docs.yaml

```diff
@@ -1,4 +1,5 @@
-name: deploy-docs
+---
```

Comment:
btw, what is this `---` for? Is it useful only if I want to cat multiple yamls to single file?

Reply:
We are using yamllint and in default. So '---' it's required for document-start. But anyway it's a good habit for YAML from my point of view. 

More info:
* https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.document_star